### PR TITLE
{compiler,tools}[system/system] Go v1.22.1, git-lfs v3.5.1

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/git-lfs/git-lfs-3.5.1.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+version = '3.5.1'
+
+homepage = 'https://git-lfs.github.com'
+description = """Git Large File Storage (LFS) replaces large files such as audio
+ samples, videos, datasets, and graphics with text pointers inside Git, while 
+ storing the file contents on a remote server like GitHub.com"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/git-lfs/git-lfs/archive/']
```
Comment (casparvl) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-3.5.1.eb:13]: Any reason for not keeping the `GITHUB_SOURCES` URL combined with `github_account=name` that's in prior EasyConfigs for `git-lfs`?

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen3

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/6af27ce0e9b76df40116d8b3929b1197 for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/g/git-lfs/git-lfs-3.5.1.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.5.1.eb]: good point, changed.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download"

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/59f3fec91ada09037af0c73147005fda for a full test report.

**Review by casparvl — APPROVED** [review]

**Comment by casparvl** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**